public/contribute.jade

```diff
@@ -23,7 +23,14 @@
 
     p Our goal is to deliver a lean, lightweight set of Angular-based UI elements that implement the material design specification for use in Angular single-page applications (SPAs).
 
-    a(href="https://github.com/angular/material/blob/master/docs/guides/CONTRIBUTING.md" class="button" md-button) Contribute to Angular Material
+    a(href="https://github.com/angular/material/blob/master/CONTRIBUTING.md" class="button" md-button) Contribute to Angular Material
+
+  .l-sub-section
+    h3 Angular Material2
+
+    p Angular team's Material Design components built on top of Angular 2. Things are in very early stages now- check back soon for more progress.
```

Comment:
I would rather say `Material Design Components, built on top of Angular 2 by the Angular Team is providing a huge amount of components using the official material design specification`.

But I'm not sure (I'm not a native speaker) - This should be reviewed by someone of the Angular team